docs/contributing-to-developer-documentation.md

```diff
@@ -396,8 +396,7 @@ one of the following topics.
 
 * [ ] Multisig wallets like GreenAddress.it
 * [x] Hardware wallets like Trezor
-* [ ] Assurance contracts such as those created by [Lighthouse](http://blog.vinumeris.com/2014/05/17/lighthouse/)
-    * [ ] Mention that the recipients of assurance contracts can pay to their own contract to guarantee that they meet their goal and receive the funds.
+* [ ] Mention that the recipients of assurance contracts can pay to their own contract to guarantee that they meet their goal and receive the funds.
```

Comment:
I think this line that was a sub-point should also be removed, as it doesn't make sense to describe a vulnerability without describing the protocol that contains that vulnerability.